### PR TITLE
Fix twitch webhooks

### PR DESCRIPTION
**Pull request by NyCodeGHG**

### CODE REVIEW

**Comment by warriorzz** [comment]
D:

**Review by warriorzz — APPROVED** [review]

File: src/main/kotlin/de/nycode/bankobot/twitch/TwitchIntegration.kt [hunk 1]
```diff
@@ -95,13 +99,24 @@ private suspend fun Kord.subscribe(userId: Int, token: TwitchAccessTokenResponse
...
 ) = launch {
-    for (unit in ticker(1.days.toLongMilliseconds())) {
-        BankoBot.httpClient.updateSubscription(userId, "subscribe", token)
-        println("Updated subscription!")
+    val duration = if (Config.ENVIRONMENT == Environment.PRODUCTION) {
+        1.days.toLongMilliseconds()
+    } else {
+        30.seconds.toLongMilliseconds()
+    }
+    for (unit in ticker(delayMillis = duration, initialDelayMillis = duration)) {
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/twitch/TwitchIntegration.kt:112]:
```suggestion
    val duration = if (Config.ENVIRONMENT == Environment.PRODUCTION) {
        1.days
    } else {
        30.seconds
    }
    val delay = duration.toLongMilliseconds()
    for (unit in ticker(delayMillis = delay , initialDelayMillis = delay)) {
```


Probably do something like this I don't like the two calls to toLongMilliseconds

File: src/main/kotlin/de/nycode/bankobot/twitch/WebhookServer.kt [hunk 2]
```diff
@@ -48,12 +51,22 @@ internal fun Kord.launchServer() = embeddedServer(CIO) {
     install(Routing) {
         route("twitch") {
             get {
+                webhookLogger.info(call.parameters.toString())
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/twitch/WebhookServer.kt:54]:
```suggestion
                webhookLogger.info { "${call.parameters}" }
```

do it like this so it only calls toString if info is enabled

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/nycode/bankobot/twitch/TwitchIntegration.kt (hunk 1 above)
Comment (NyCodeGHG) [reply src/main/kotlin/de/nycode/bankobot/twitch/TwitchIntegration.kt:112]: 👍

File: src/main/kotlin/de/nycode/bankobot/twitch/WebhookServer.kt (hunk 2 above)
Comment (NyCodeGHG) [reply src/main/kotlin/de/nycode/bankobot/twitch/WebhookServer.kt:54]: I forgot to remove those debug messages

**Review by DRSchlaubi — APPROVED** [review]

**Pull request merged**